src/components/RouterCourse/Courses/index.js

```diff
@@ -0,0 +1,47 @@
+import React from 'react';
+import { NavLink } from 'react-router-dom';
+
+import CourseContainer from './CourseContainer';
+import { HTMLCourses, CSSCourses, JSCourses } from '../data/Courses';
+
+const Courses = ({course}) => {
+  const courseDataMap = {
+    html: HTMLCourses,
+    css: CSSCourses,
+    javascript: JSCourses,
+  };
```

Comment:
The `courseDataMap ` can be placed outside the function component so it won't be redeclared every time the component re-renders.

Then, we can also implement implicit return in this component.

Reply:
updated, code change are now applied